src/pages/index.tsx

```diff
@@ -1,19 +1,9 @@
-import styles from "@/styles/Home.module.css";
-import { Inter } from "next/font/google";
-import Head from "next/head";
-import Image from "next/image";
-import styled from "styled-components";
-
-const inter = Inter({ subsets: ["latin"] });
-
-const StyledDiv = styled.div`
-  color: red;
-`;
+import Link from "next/link";
 
 export default function Home() {
   return (
     <>
-      <StyledDiv>Styled Div</StyledDiv>
+      <Link href={"/rooms/all"}> go to rooms</Link>
```

Comment:
What I usually tend to do is have e.g. an object mapping all the navigation paths, because you usually will link to certain pages (e.g. the room overview) from different parts in your app, but then you'll have to updatethe `"/rooms/all"` string everywhere manually




```typescript
// routes.ts (top level in src)
export const ROUTES = {
  "ROOMS_OVERVIEW": "/rooms/all"
}
```

and then you can use

```suggestion
      <Link href={ROUTES.ROOMS_OVERVIEW}> go to rooms</Link>
```

Reply:
That's such a good idea, never thought about that. This will make my life SO much easier, because i always forget to update the path strings